Deprecations now include Win2008 (#1108)
OctopusDeploy/blog: blog/2023-08/deprecating-win2003/index.md

|Windows 2012 R2|.NET4.5.1|.NET4.8+|
|Windows 2016|.NET4.6.2|.NET4.8+|

> Note that as shown above, the latest service pack will be required to be installed on Windows Server 2008 operating systems to allow for the installation of .NET4.6.2

Reviewer: ```suggestion
Note that as shown above, you'll need the latest service pack installed on Windows Server 2008 operating systems to allow for the installation of .NET4.6.2.
```
